packages/forms/src/UIForm/fields/Code/Code.component.test.js

```diff
@@ -113,8 +118,7 @@ describe('Code field', () => {
 
 	it('should manage tabindex on textarea', () => {
 		// given
-		const wrapper = mount(<Code.WrappedComponent {...props} />);
-		const container = wrapper.find('#myCodeWidget-editor-container');
+		const wrapper = mount(<Code {...props} />, { attachTo: document.body });
```

Comment:
Need to mount on document.body with talend/script jest/enzyme version, or the document.activeElement is not set.